modules/multi-architecture-scheduling-workloads.adoc

```diff
@@ -0,0 +1,58 @@
+// module included in the following assembly 
+//
+//post_installation_configuration/configuring-multi-arch-compute-machines/multi-architecture-compute-managing.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-scheduling-workloads_{context}"]
+
+= Creating a RuntimeClass for scheduling mechanisms 
+
+Using a `RuntimeClass` object simplifies the use of scheduling mechanisms, including taints and tolerations. You can apply a runtime class to your pod and schedule them to your ARM64 or AMD64 compute node. 
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`)
+* You created an ARM64 compute machine set on your cluster
+
+.Procedure 
+. Create a `RuntimeClass` object with the following command:
++
+-- 
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+.Example `RuntimeClass` object:
+[source,yaml]
+----
+apiVersion: node.k8s.io/v1beta1
+kind: RuntimeClass
+metadata:
+  name: <runtime_class_name> <1>
+handler: '' //not sure what these values are 
+scheduling:
+  nodeSelector: <2>
```

Comment:
maybe a short description of the fact that we need nodeSelector's to ensure that the pod using the runtimeclass lands on the nodes that support it.